velox/exec/tests/PlanBuilderTest.cpp

```diff
@@ -250,4 +254,60 @@ TEST_F(PlanBuilderTest, missingOutputType) {
       "outputType must be specified");
 }
 
+TEST_F(PlanBuilderTest, projectExpressions) {
+  // Non-typed Expression
+  auto data = ROW({"c0"}, {BIGINT()});
+  VELOX_CHECK_EQ(
+      PlanBuilder()
+          .tableScan("tmp", data)
+          .projectExpressions(
+              {std::make_shared<core::FieldAccessExpr>("c0", std::nullopt)})
+          .planNode()
+          ->toString(true, true, nullptr),
+      PlanBuilder()
+          .tableScan("tmp", data)
+          .project({"c0"})
+          .planNode()
+          ->toString(true, true, nullptr)); // simple field access
```

Comment:
nit: You can use a string as the expected value. See the examples in TEST_F(PlanBuilderTest, windowFunctionCall). Same for the check below.